contents/docs/error-tracking/installation/hono.mdx

```diff
@@ -21,21 +21,19 @@ npm install --save posthog-node
 Hono uses [`app.onError`](https://hono.dev/docs/api/exception#handling-httpexception) to handle uncaught exceptions. You can take advantage of this for error tracking. Remember to **export** your [project API key](https://app.posthog.com/settings/project#variables) as an environment variable.
 
 ```ts file=index.ts
-import { env } from 'hono/adapter'
 import { PostHog } from 'posthog-node'
 
-app.onError((err, c) => {
-  const { POSTHOG_PUBLIC_KEY } = env<{ POSTHOG_PUBLIC_KEY:string }>(c)
-  const posthog = new PostHog(POSTHOG_PUBLIC_KEY, { host: '<ph_client_api_host>' })
-  
-  posthog.captureException(new Error(err.message, { cause: err }), 'user_distinct_id_with_err_rethrow', {
+const posthog = new PostHog(process.env.POSTHOG_PUBLIC_KEY, { host: '<ph_client_api_host>' })
+
+app.onError(async (err, c) => {
+  posthog.captureException(err, 'user_distinct_id_with_err_rethrow', {
     path: c.req.path,
     method: c.req.method,
     url: c.req.url,
     headers: c.req.header(),
     // ... other properties
   })
-  posthog.shutdown() // TIP: On program exit, call shutdown to stop pending pollers and flush any remaining events
+  await posthog.flush() // TIP: On program exit, call shutdown to stop pending pollers and flush any remaining events
```

Comment:
fix the comment? you tell to call shutdown but call flush
or do posthog.flush and then shutdown